test/detail/BitStreamTests.cpp

```diff
@@ -117,7 +117,7 @@ TEST_CASE("BitStream") {
            const auto n = *gen::inRange(0, 64);
            const bool sign = (x & (1LL << (n - 1LL))) != 0;
```

Comment:
Yeah, you're right. The test looks wrong. Zero bits will never yield a signed value so the signed path should never be executed in that case. However, I'd prefer it if you made `sign` actually be false in that case here since that'd be more consistent.